components/MnemonicView.tsx

```diff
@@ -141,14 +157,16 @@ const MnemonicView: FC<MnemonicViewProps> = ({
         {$currentWords.map((word, index) => {
           return (
             <MnemonicInput
-              key={index}
+              key={`mnemonic-phrase-view-item-${index}`}
+              index={index}
```

Comment:
Use combination `word+index` for key.

Reply:
will keep used key, as its possible to input words so `word` will always change.